pyproject.toml

```diff
@@ -19,17 +19,13 @@ esdbclient = {version = ">=1.0b3", optional = true}
 django = {version = ">=4,<6", optional = true}
 
 [tool.poetry.group.dev.dependencies]
-black = "*"
-isort = "*"
-flake8 = "*"
-bandit = "*"
```

Comment:
this should stay and be plugged into pipeline if it wasn't used so far. It's supposed to check for vulnerabilities

Reply:
I'm withdrawing this comment, as I later found out "B" for ruff includes bandit rules